### ISSUE TITLE
Unable to start an onChange event on a model's relation field

### ISSUE THREAD

**Issue by swappsshinde** [strapi/strapi#2995]
**Informations**

Node.js version: v10.15.0
NPM version: 6.7.0
Strapi version: 3.0.0-alpha.24.1
Database: MySQL
Operating system: Ubuntu 18.04
(Optional) Link to your Project:

**What is the current behaviour?**
There are two relation fields in my ProductCombinationAttribute table 
1. Attribute Type:(Dropdown on this relation field) Color, Size
2. Attribute Value:(Dropdown on this relation field) The problem comes here Attribute value field gives me all the options -- Gold, Silver, 64Gb, 128Gb -- Whereas what I want is that Attribute value field gives me values in the dropdown on the basis of Attribute type Field

**Steps to reproduce the problem**
Create 3 content types: 
1. Attribute: Which has a String-name -- {Color, Size}
2. Attributevalue: Which has String-Attribute value, along with a relation ( Attribute has many Attribute Values) -- {Gold, relation-Color || Silver, relation-Color || 64Gb, relation-Size || 256Gb, relation-Size}
3. Productcombinationattribute - Which has  Boolean-Enabled, along with 2 relations:
    A> Attribute has many Productcombinationattributes
    B> Attributevalue has many Productcombinationattributes

**What is the expected behaviour?**

What I want is that in the _Productcombinationattribute_  content type on selecting the  _Attribute_  relation the values in _Attributevalue_ (Gold, Silver, 64Gb, 256Gb) change on the basis of value in _Attribute_ (Color, Size). 

So, basically when I select _Attribute_ as (Color) the fields in the dropdown of _Attributevalue_ should only give (Gold, Silver)

**Suggested solutions**
I am pretty new to this environment and I am unable to understand the complexities of the code. A thorough explanation of my problem would be much appreciated. Thank You In advance :)

@lauriejim  @Aurelsicoko

**Comment by lauriejim** [comment]
Hello @swappsshinde !

I think you have to create 2 different content type, one for your colors and one for your sizes.
Your data structure is not adapted to your need.

**Comment by lauriejim** [comment]
Hum customise the content manager plugin is really not an easy way. I did not recommend to customise it without learn how it works.

Maybe the best solution for you is to create `enum` attributes instead of content type.
